src/pages/SubmitWorkflow.js

```diff
@@ -11,14 +13,37 @@ export const SubmitWorkflow = () => {
   const [workflowInputs, setWorkflowInputs] = useState()
   const [showInputsPage, setShowInputsPage] = useState(false)
 
+  const signal = useCancellation()
+
+  const submitRun = async () => {
+    const runRes = await Ajax(signal).Cbas.submitRun(workflowUrl, workflowInputs)
+    console.log(runRes)
+  }
+
   return div([
     headerBar(),
     div({ style: { margin: '4rem' } }, [
       h2(['Submit a workflow']),
       div(['Submit your Terra workflows with the Cromwell engine. Full featured workflow submissions coming soon!']),
       div({ style: { marginTop: '2rem' } }, [
-        !showInputsPage && h(WorkflowSource, { workflowUrl, setWorkflowUrl, onClick: () => setShowInputsPage(true) }),
-        showInputsPage && h(WorkflowInputs, { workflowUrl, workflowInputs, setWorkflowInputs, displayWorkflowUrlPage: () => setShowInputsPage(false) })
+        !showInputsPage && h(WorkflowSource, { workflowUrl, setWorkflowUrl }),
+        !showInputsPage && div({ style: { display: 'flex', justifyContent: 'flex-end', marginTop: '1rem' } }, [
```

Comment:
I have a suspicion you can group these together in an array, eg `!showInputsPage && [ h(...), div(...) ]`

Reply:
Today I re-learned about `Fragments` that can be used to group different components/tags. Simply using `[..]` throws error that unique key needs to be assigned to each element in array, but there is no need for that in `Fragment` hence updated it 👍